Worktrolly/worktrolly
# Dev/link

### Functionality:
Add Hierarchical Linking Mechanism and Re-write login handler to fetch Data using API

### Solution:
1. Added a Link collection under tasks to track links
2. In task details page, displayed available links 
3. Provided button to add new links
4. Added getUserAppSettings Api to get user data in authservice

### Risk level:

### How to test:
1. firebase emulators:start
2. ng serve
3. auto setup
4. in task details page on any task, create a new link
5. Observe link collection added under the task in db
